deploy/monitor/prometheus_rule.yaml

```diff
@@ -23,13 +23,13 @@ spec:
           description: "The UnifiedPush Operator has been down for more than 5 minutes. "
           summary: "The UnifiedPush Operator is down. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
           sop_url: "https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc"
-      - alert: UnifiedpushPodcount
+      - alert: UnifiedPushPodcount
         annotations:
-          description: "The Pod count for the UnifiedPush has changed in the last 5 minutes."
-          summary: "Pod count for namespace UnifiedPush is {{ printf '%.0f' $value }}. Expected 3 pods. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
+          description: "The Pod count for the UnifiedPush has changed in the last 5 minutes and is lower than the minimal quantity required."
+          summary: "Pod count for namespace UnifiedPush is {{ printf '%.0f' $value }}. Expected 3 pods at least. For more information see on the UnifiedPush Operator https://github.com/aerogear/unifiedpush-operator"
```

Comment:
This says "Pod count for namespace UnifiedPush". We don't necessarily know the name of the namespace right now. Is there a way to template this here?

Reply:
Please check the comment which follows.